src/meta/model_v2/src/lib.rs

```diff
@@ -164,7 +166,95 @@ macro_rules! derive_from_json_struct {
     };
 }
 
-pub(crate) use derive_from_json_struct;
+/// Defines struct with a byte array that derives `DeriveValueType`, it will helps to map blob stored in database to Pb struct.
+macro_rules! derive_from_byte_array {
```

Comment:
add a helper macro to map `blob` column type to Pb struct.

Reply:
```suggestion
macro_rules! derive_from_blob {
```

Can we name it `blob`? It could be more clear to me.